ext/mini_racer_extension/mini_racer_extension.cc

```diff
@@ -1,8 +1,8 @@
 #include <stdio.h>
 #include <ruby.h>
 #include <ruby/thread.h>
-#include <v8.h>
-#include <libplatform/libplatform.h>
+#include <include/v8.h>
+#include <include/libplatform/libplatform.h>
```

Comment:
not sure why is "include/" necessary here, maybe @cowboyd could chime with an opinion if it's OK to change the include paths in libv8